src/graphics/graphics-device.js

```diff
@@ -75,6 +75,12 @@ class GraphicsDevice extends EventHandler {
      */
     precision;
 
+    /** @type {RenderTarget} */
```

Comment:
perhaps an explanation here on how this member is used by render-pass system?

Reply:
this is an existing member, but it was never declared and just assigned to in few places, so I added it as a member with type. It's not directly used by render passes. I'll add comment stating it's currently assigned render target.